my-element.js

```diff
@@ -35,28 +37,64 @@ export class MyElement extends LitElement {
        * The number of times the button has been clicked.
        */
       count: {type: Number},
+
+      /**
+       * The array to store table entries
+       */
+      data: {type: Array}
     };
   }
 
   constructor() {
     super();
     this.name = 'World';
     this.count = 0;
+    this.data = [
+      {id:1 , well: 'A1', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}},
+      {id:2 , well: 'A2', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}},
+      {id:3 , well: 'A4', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}},
+      {id:4 , well: 'A4', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}},
+      {id:5 , well: 'A4', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}},
+      {id:6 , well: 'A4', library:'MC03', previousPlate:'', previousCoordinate:'', feature:'', nrxId:'NRX-0252942', barcode:'NUR900149', target:'ABC', group:'',validation:{eip:'142123',hp:'HP-009',val:91}}
+    ];
+    this.handleEdit = this.handleEdit.bind(this);
+        this.handleDelete = this.handleDelete.bind(this);
+        this.handleKebabMenuClick = this.handleKebabMenuClick.bind(this);
   }
 
   render() {
     return html`
-      <h1>Hello, ${this.name}!</h1>
-      <button @click=${this._onClick} part="button">
-        Click Count: ${this.count}
-      </button>
-      <slot></slot>
+    <app-menu .onEdit="${this.handleEdit}" .onDelete="${this.handleDelete}"></app-menu>
+      <app-table .data="${this.data}" .onKebabMenuClick="${this.handleKebabMenuClick}"></app-table>
+    
     `;
   }
 
   _onClick() {
     this.count++;
   }
+
+
+  handleEdit(id) {
+    // console.log('edit', id);
+    this.shadowRoot.querySelector('app-menu').hidden = true;
```

Comment:
lets not do the dom manipulation. INstead use a boolean prop and based on the prop display the component like:
```
${ this.isAppMenuOpen ? <app-menu> : nothinh}
```